testing/gsoap/APKBUILD

```diff
@@ -0,0 +1,62 @@
+# Contributor: <xmingske@gmail.com>
+# Maintainer: Natanael Copa <ncopa@alpinelinux.org>
```

Comment:
hmm, I've asked him once if he would mind if I put him as Maintainer in the APKBUILDs and he said no at that time, did the policy change?